platform/testing/liquid-templates/README.md

```diff
@@ -5,17 +5,172 @@ The Liquid Template Unit Testing Framework was created to replace Cypress for un
+    <!-- lots of liquid code here -->
+  </nav>
+</body>
+</html>
+```
+
+Just like a browser, `JSDOM` adds whatever `HTML` is missing to make the document valid.
+
+Notice there's no `title` tag in the `head` of the document. Because of this, we can't perform Axe Checks to make sure a `title` is present, so we need to disable the `document-title` check.
+
+In addition to the reason outlined above, we experienced false-positives on pages that had a valid `title` tag when running the `document-title` check. 
+
+If you want to test the presence of a `title` tag, and that the `title` isn't an empty string, you'll have write unit tests for those 'the old fashion way'.
+
+#### How to Run an Axe Check
+
+Run an accessibility check like this:
+
+```js
+it('reports no axe violations', async () => {
+  const violations = await axeCheck(container);
+  expect(violations.length).to.equal(0);
+});
+```
+
+Accessibility violations are logged to the console and look like this:
+
+<details>
+  <summary>Accessibility Violations</summary>
+
+  ```javascript
+  6 Accessibility Violations Were Detected
+
+  Axe Violation 1:
+  {
+    id: 'aria-roles',
+    impact: 'critical',
+    tags: [ 'cat.aria', 'wcag2a', 'wcag412' ],
+    description: 'Ensures all elements with a role attribute use a valid value',
+    help: 'ARIA roles used must conform to valid values',
+    helpUrl: 'https://dequeuniversity.com/rules/axe/4.1/aria-roles?application=axeAPI',
+    nodes: [
+      {
+        any: [],
+        all: [],
+        none: [Array],
+        impact: 'critical',
+        html: '<h1 role="nonsense">VA Pittsburgh health care</h1>',
+        target: [Array],
+        failureSummary: 'Fix all of the following:\n' +
+          '  Role must be one of the valid ARIA roles: nonsense'
+      }
+    ]
+  }
 
-This technique can be used to generate tests of varying complexity, ranging from simple rendering sanity checks to complex logic. Since we control the JSON test data, we can easily test different scenarios.
+  Node 1:
+  {
+    any: [],
+    all: [],
+    none: [
+      CheckResult {
+        id: 'invalidrole',
+        data: [Array],
+        relatedNodes: [],
+        impact: 'critical',
+        message: 'Role must be one of the valid ARIA roles: nonsense'
+      }
+    ],
+    impact: 'critical',
+    html: '<h1 role="nonsense">VA Pittsburgh health care</h1>',
+    target: [ 'h1' ],
+    failureSummary: 'Fix all of the following:\n' +
+      '  Role must be one of the valid ARIA roles: nonsense'
+  }
+  ```
+</details>
+
+#### `axe-core` Smoke Tests
+A smoke test was created to ensure `axe-core` is picking up the expected violiations.
+
+The liquid template for the smoke test is located at `src/site/layouts/liquid_template_axe_check_smoke_test.drupal.liquid`. The spec file and fixture is in the `src/site/layouts/tests/liquid_template_axe_check_smoke_test` directory.
```

Comment:
@holdenhinkle What do you think about adding a sentence or two stating this test should pick up 6 violations intentionally to pass? I think mentioning that this test acts as a circuit breaker for potential silent failures is an important detail.

Reply:
Good idea. I just updated it.